source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -10,10 +10,22 @@ content: |
      - **access key ID**
      - **secret access key**
 
-  2. Grant the IAM user narrowly scoped ``List`` and ``Read`` permissions only for the KMS
-     service. See Amazon's official documentation on
-     `Adding permissions to a user <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_change-permissions.html>`__
-     to set these permissions.
+  2. Grant your IAM user ``kms:Encrypt`` and ``kms:Decrypt`` permissions for
+     your remote master key. The following inline 
+     policy allows an IAM user to encrypt and decrypt with the remote master key
+     with the least privileges possible:
+
+     .. literalinclude:: /includes/iam-user-policy-minimum.json
+        :language: json
+
+     .. note:: Remote Master Key Amazon Resource Name (ARN)
+
+        The preceding policy requires the ARN of the key you generate in the
+        :ref:`Create the Master Key <aws-create-master-key>` step of this guide.
+
+     To learn how to set this permission, see
+     `Changing permissions for an IAM user <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_change-permissions.html>`__
+     from AWS.
```

Comment:
Issue: it's not clear what "from AWS" means.
Suggestion: clarify what the link refers to (e.g. "see the AWS documentation on Changing permissions..." or omit it.

Reply:
Changed "from AWS" to "in the official AWS documentation". I believe the "To learn more, see <title of other page> in <destination>" is the preferred structure of external links at the end of sections in the [style guide](https://docs.mongodb.com/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text), so I think it's best to keep this structure. 

Relevant items from [links and cross references](https://docs.mongodb.com/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text) entry of the style guide:

> For links at the end of an article or topic that point to related information or to a next step, use the title of or a heading in the target content as the link text.

> If a link points to a location other than the current site (for example, out of the Support website or away from docs.mongodb.com), provide context that describes the location.